web/themes/custom/rebartik/src/scss/pages/home/_mixin.scss

```diff
@@ -0,0 +1,135 @@
+@import '../../_global';
```

Comment:
Why do you plan to use the '_mixin.scss' separatelly from '_global.scss'?

Generally, we have to place mixins next to variables and include them in global.